vdsm.spec.in

```diff
@@ -499,6 +499,14 @@ The system administrator will be responsible for creating the host "ovirt-local"
 volume group and extending it with new devices if needed.
 The VM must be pinned to the host.
 
+%package hook-log_console
```

Comment:
I would name it hook-log-console (with dashes, no underscore), see hook-ethtool-options.

Reply:
Done